Private/Find-ESC4.ps1

```diff
@@ -46,7 +46,8 @@
         [Parameter(Mandatory = $true)]
         $SafeUsers,
         [Parameter(Mandatory = $true)]
-        $SafeObjectTypes
+        $SafeObjectTypes,
+        $Mode
```

Comment:
LGTM. Do you have time to update the CBH real quick while your in the..."mode?" 😜

Reply:
By Thursday's webcast, I promise!